src/main/java/com/patina/codebloom/common/db/repos/api/access/ApiKeyAccessSqlRepository.java

```diff
@@ -0,0 +1,191 @@
+package com.patina.codebloom.common.db.repos.api.access;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Set;
+import java.util.UUID;
+
+import org.springframework.stereotype.Component;
+
+import com.patina.codebloom.common.db.DbConnection;
+import com.patina.codebloom.common.db.helper.NamedPreparedStatement;
+import com.patina.codebloom.common.db.models.api.access.ApiKeyAccess;
+@Component
```

Comment:
nit: spacing between the two

```suggestion
import com.patina.codebloom.common.db.models.api.access.ApiKeyAccess;

@Component
```